ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -181,6 +188,45 @@ export const displayModuleExpirationDate = (
     getNullStringForExpirationDate(moduleName)
   );
 
+const isModuleRequiredByAccessTier = (
```

Comment:
this function only deals with ERA so its name should reflect that

Reply:
I was making this func generic and can be used to handle new or existing special modules in future. Do you agree?

Reply:
no - ERA is very special and weird.  We don't expect to have any future modules like it.

Reply:
Done. Renamed to `isEraRequiredForTier`.